components/curriculum/vars.css

```diff
@@ -209,26 +215,53 @@
-    --curriculum-landing-started-employment: url("./assets/curriculum-landing-started-employment.svg#dark");
-    --curriculum-landing-started-educator: url("./assets/curriculum-landing-started-educator.svg#dark");
-    --curriculum-module-mdn-resource: url("./assets/curriculum-mdn-resource.svg#dark");
-    --curriculum-bullet: url("./assets/curriculum-bullet.svg#dark");
-    --curriculum-about-covered: url("./assets/curriculum-about-covered.svg#dark");
-    --curriculum-about-detail: url("./assets/curriculum-about-detail.svg#dark");
-    --curriculum-about-educators: url("./assets/curriculum-about-educators.svg#dark");
-    --curriculum-about-not: url("./assets/curriculum-about-not.svg#dark");
-    --curriculum-about-students: url("./assets/curriculum-about-students.svg#dark");
-    --curriculum-bg-image-partner: url("./assets/curriculum-partner-bg.svg#dark");
-    --curriculum-scrim-bg: url("./assets/curriculum-scrim-bg.svg#dark");
-    --curriculum-landing-arrow: url("./assets/curriculum-landing-arrow.svg#dark");
+:root[data-theme="light dark"] {
+  @media (prefers-color-scheme: dark) {
+    .curriculum-content-container {
+      --test-theme: #0000ff;
+      --curriculum-landing-about-beginner: url("./assets/curriculum-landing-about-beginner.svg#dark");
+      --curriculum-landing-about-pace: url("./assets/curriculum-landing-about-pace.svg#dark");
+      --curriculum-landing-about-free: url("./assets/curriculum-landing-about-free.svg#dark");
+      --curriculum-landing-about-bullet: url("./assets/curriculum-landing-about-bullet.svg#dark");
+      --curriculum-landing-started-beginner: url("./assets/curriculum-landing-started-beginner.svg#dark");
+      --curriculum-landing-started-advanced: url("./assets/curriculum-landing-started-advanced.svg#dark");
+      --curriculum-landing-started-employment: url("./assets/curriculum-landing-started-employment.svg#dark");
+      --curriculum-landing-started-educator: url("./assets/curriculum-landing-started-educator.svg#dark");
+      --curriculum-module-mdn-resource: url("./assets/curriculum-mdn-resource.svg#dark");
+      --curriculum-bullet: url("./assets/curriculum-bullet.svg#dark");
+      --curriculum-about-covered: url("./assets/curriculum-about-covered.svg#dark");
+      --curriculum-about-detail: url("./assets/curriculum-about-detail.svg#dark");
+      --curriculum-about-educators: url("./assets/curriculum-about-educators.svg#dark");
+      --curriculum-about-not: url("./assets/curriculum-about-not.svg#dark");
+      --curriculum-about-students: url("./assets/curriculum-about-students.svg#dark");
+      --curriculum-bg-image-partner: url("./assets/curriculum-partner-bg.svg#dark");
+      --curriculum-scrim-bg: url("./assets/curriculum-scrim-bg.svg#dark");
+      --curriculum-landing-arrow: url("./assets/curriculum-landing-arrow.svg#dark");
+    }
+  }
+
+  @media (prefers-color-scheme: light) {
```

Comment:
Could we avoid this by changing `:root[data-theme="light"] {` above as simply `:root {`?

Reply:
There is some weird interaction between color-scheme and the media queries.

Reply:
I'll keep it as is, will check for a better solution later.